src/datasets/features/image.py

```diff
@@ -226,12 +92,14 @@ def decode_example(self, value):
             raise ImportError("To support decoding images, please install 'Pillow'.")
 
         if isinstance(value, str):
-            path, bytes_ = value, None
+            path, bytes_ = value, None  # TODO(QL, Mario): remove this case ?
```

Comment:
Yes, let's remove this because it's no longer needed. Previously, we would need this for the hacky `cast_column` cast from `str` to `PIL.Image`:
```python
ds = Dataset.from_dict({"image": ["path/to/image1", "path/to/image2"]})
ds = ds.cast_column("image", Image())
```